WebDevStudios/constant-contact-forms
# CC-135: Address CTCT block errors

Ticket: [CC-135](https://webdevstudios.atlassian.net/browse/CC-135)

## Description ##

Updates CTCT block to use `wp` destructuring instead of `import` to avoid "Invalid Hook Call Warning" errors.
Removes now-unnecessary WP dev dependencies.

## Testing ##

- Add "Constant Contact: Single Form" block to a page.
- Confirm no fatal errors appear in console.
- Confirm block works as expected:
    - Dropdown appears populated with CTCT forms.
    - Once form is selected and page saved, appropriate form appears on the frontend.
